heron/examples/src/java/com/twitter/heron/examples/CustomGroupingTopology.java

```diff
@@ -18,17 +18,17 @@
 import java.util.List;
 import java.util.Map;
 
-import backtype.storm.Config;
-import backtype.storm.StormSubmitter;
-import backtype.storm.generated.GlobalStreamId;
-import backtype.storm.grouping.CustomStreamGrouping;
-import backtype.storm.task.OutputCollector;
-import backtype.storm.task.TopologyContext;
-import backtype.storm.task.WorkerTopologyContext;
-import backtype.storm.topology.OutputFieldsDeclarer;
-import backtype.storm.topology.TopologyBuilder;
-import backtype.storm.topology.base.BaseRichBolt;
-import backtype.storm.tuple.Tuple;
+import org.apache.storm.Config;
```

Comment:
I agree that we should not mix backtype api (which is storm < 1.0) and org.apache.storm api (storm >= 1.0) together.